components/FeedItem/FeedItem.tsx

```diff
@@ -0,0 +1,41 @@
+import RSSParser from "rss-parser";
+import { Feed } from "../../models";
+import { DateComponent } from "../Date/Date";
+import { ExternalLink } from "../ExternalLink/ExternalLink";
+import Styles from "./FeedItem.module.css";
+
+type Props = {
+  item: RSSParser.Item;
+  feed: Feed;
+  onClick?: () => void;
+  onFavoriteClick?: () => void;
+  isFavorited?: boolean;
+};
+
+export function FeedItem({
+  item,
+  feed,
```

Comment:
`yarn lint` failed in CI.

This prop is not used, let's remove it from `Props` also.

Reply:
You forgot this part😅
> let's remove it from Props also.